src/test/all_downloads.ml

```diff
@@ -25,7 +25,41 @@ let run_program ?name ?requirements prog =
   let open Ketrew.EDSL in
   daemonize ~using:`Python_daemon ~host prog
 
-let workflow =
+
+let biocaml_def = Biokepi.Machine.Tool.Definition.create "biocaml" ~version:"0.4.0"
+let biocaml =
+  Biokepi.Setup.Biopam.install_target biocaml_def
+    ~tool_type:(`Library "biocaml")
+    ~witness:"META"
+    ~repository:`Opam
+    ~compiler:"4.02.3"
+
+let ledit_def = Biokepi.Machine.Tool.Definition.create "ledit" ~version:"2.03"
```

Comment:
An ocaml program that installs very fast with any ocaml compiler (and I happen to still prefer `ledit ocaml` rather than `utop` ☺):

```
 $ opam info ledit
             package: ledit
             version: 2.03
          repository: default
        upstream-url: http://pauillac.inria.fr/~ddr/ledit/distrib/src/ledit-2.03.tgz
       upstream-kind: http
   upstream-checksum: 3a70ee7d5d5e2dfb905a1ac2e1e60276
             depends: camlp5
   installed-version: 2.03 [4.02.3+smondet-fix7013]
   available-version: 2.03
         description: Line editor, a la rlwrap
One-line editor written in OCaml. It provides line editing for the
Caml toplevels, as well as other interactive Unix commands.
```